genconvert/input/my-t-my-d0-zawgyi.txt

```diff
@@ -36,8 +36,8 @@ $vowelmedial = [\u102B-\u1030\u1032\u103c-\u103F];
 $ukinzi = [\u1004\u101b\u105a]\u103A\u1039;
 
 $medialraZ = [\u103b\u107e-\u1084];
-$lowsignZ = [\u102f\u1030\u1037\u103a\u103c\u103d\u1087-\u108a];
```

Comment:
Z-103a has a descender that should be considered in picking the medial ra code. I think this should stay in lowsignZ.

Reply:
Actually Z-103A is also the ra with different usage. There is no usage combining those two.